pkg/ingress/controller/apisix_route.go

```diff
@@ -232,10 +230,15 @@ func (c *ApisixRouteController) sync(rqo *RouteQueueObj) error {
 		rc := &state.RouteCompare{OldRoutes: oldRoutes, NewRoutes: newRoutes}
 		return rc.Sync()
 	case rqo.Ope == DELETE:
-		apisixRoute := apisix.ApisixRoute(*apisixIngressRoute)
+		apisixIngressRoute, _ := c.apisixRouteList.ApisixRoutes(namespace).Get(name)
+		if apisixIngressRoute != nil && apisixIngressRoute.ResourceVersion > rqo.OldObj.ResourceVersion {
```

Comment:
We'd better to log it, it's somewhat confusing (if we don't know the retry knowledge background).

Reply:
Done, add a warn level log